quickstarts/browser/config.yml

```diff
@@ -43,3 +43,6 @@ dashboards:
   - browser-pages-dashboard
   - javascript-errors
   - traffic-analysis
+alertPolicies:
+  - browser
+  - ajax-example
```

Comment:
Seems like this alert policy doesn't exist. Either remove it or create an alert with this name.